Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/TodoConfiguration.swift

```diff
@@ -0,0 +1,27 @@
+import SwiftLintCore
+
+struct TodoConfiguration: SeverityBasedRuleConfiguration, Equatable {
+    typealias Parent = TodoRule
+
+    @ConfigurationElement(key: "severity")
+    private(set) var severityConfiguration = SeverityConfiguration<Parent>(.warning)
+    @ConfigurationElement(key: "only")
+    private(set) var onlyKeywords: Set<String> = [
+        "FIXME",
+        "TODO"
```

Comment:
Better to use an enum for these specific values.

Reply:
Thank you, I fixed it!